packages/engine/src/common/functions/logger.ts

```diff
@@ -29,21 +29,29 @@ Ethereal Engine. All Rights Reserved.
  *
  * All log events (info, warn, error, etc) are:
  *  1. Printed to the console (likely browser console), and (optionally)
- *  2. Sent to the server-side /api/log endpoint, where the pino logger
+ *  2. Sent to the server-side logs-api service, where the pino logger
  *     instance (see packages/server-core/src/logger.ts) sends them to Elastic etc.
  *     Note: The sending/aggregation to Elastic only happens when APP_ENV !== 'development'.
  *
  */
 
-import fetch from 'cross-fetch'
-import LruCache from 'lru-cache'
+import { ServiceTypes } from '@etherealengine/common/declarations'
+import config from '@etherealengine/common/src/config'
+import { FeathersApplication } from '@feathersjs/feathers'
+import { logsApiPath } from '../../schemas/cluster/logs-api.schema'
 
-import config from './config'
+// const logRequestCache = new LruCache({
```

Comment:
@HexaField so we have fixed client logs being sent to server and opensearch. But by looking into the number of requests being sent from client to `logs-api` service, I think we should have some sort of caching on client end.